packages/core/src/components/loading-skeleton/loadingSkeleton.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as React from "react";
+
+import { AbstractComponent, Classes, IProps } from "../../common";
+
+export interface ILoadingSkeletonProps extends IProps {
+    /**
+     * If true, show an animated loading skeleton when `isLoading` is true. Otherwise show a static
+     * skeleton.
+     * @default true
+     */
+    animated?: boolean;
+
+    /**
+     * If true, show a loading skeleton. Otherwise render this component's child.
+     */
+    isLoading: boolean;
+
+    /**
+     * Specifies the number of animated lines to show.
+     * @default 1
+     */
+    numBones?: number;
+
+    /**
+     * If true, render a random width loading skeleton. This can be used to make things like table
+     * cells display a more natural-feeling loading state.
+     * @default false
+     */
+    randomWidth?: boolean;
+}
+
+export interface ILoadingSkeletonState {
+    animated?: boolean;
+    rightMargins?: number[];
+}
+
+export class LoadingSkeleton extends AbstractComponent<ILoadingSkeletonProps, ILoadingSkeletonState> {
+    public static defaultProps: ILoadingSkeletonProps = {
+        animated: true,
```

Comment:
I think we should remove both this prop and the corresponding CSS modifier. We don't have a use case for non-animated skeletons and I don't foresee any at the moment